lang/en_US.json

```diff
@@ -0,0 +1,12 @@
+{
+    ":count posts in the last 24 hours": ":count posts in the last 24 hours",
+    ":count posts in the last 7 days": ":count posts in the last 7 days",
+    "Additional Posts": "Additional Posts",
+    "Forum Index": "Forum Index",
+    "in": "in",
+    "Last Post By": "Last Post By",
+    "Message": "Message",
+    "Next :count": "Next :count",
+    "Previous :count": "Previous :count",
+    "Recent Posts": "Recent Posts"
+}
```

Comment:
Note that this is following a Laravel best practice, see [here](https://laravel.com/docs/11.x/localization#using-translation-strings-as-keys).

> For applications with a large number of translatable strings, defining every string with a "short key" can become confusing when referencing the keys in your views and it is cumbersome to continually invent keys for every translation string supported by your application.

Other locales would re-use these en_US keys, as that is the default locale of the project, eg:
```json
{
    ":count posts in the last 24 hours": ":count Beiträge in den letzten 24 Stunden",
    ":count posts in the last 7 days": ":count Beiträge in den letzten 7 Tagen",
    "Additional Posts": "Zusätzliche Beiträge",
    "Forum Index": "Forenindex",
    "in": "in",
    "Last Post By": "Letzter Beitrag von",
    "Message": "Nachricht",
    "Next :count": "Nächste :count",
    "Previous :count": "Vorherige :count",
    "Recent Posts": "Neueste Beiträge"
}
```